lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Result returned from the authority's token endpoint.
+ */
+export type AuthenticationResult = {
+    // TODO this is temp class, it will be updated.
+    accessToken: string;
+    refreshToken: string;
+    idToken: string;
+    expiresOn: string;
```

Comment:
Is string the right type for this?

Reply:
+1. We are returning a Javascript Date object normally. I'm also not sure why node needs a different response object from browser.

Reply:
This object is a placeholder - not 100% sure if it will be added. The difference between this and TokenResponse is that we will need to add some Account info. 

Agreed that expiresOn should probably be a Date.